skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md

```diff
@@ -287,6 +287,8 @@ Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+> [!NOTE] 
```

Comment:
Can you change this to **Note:** as the markdown doesn't render in Windows PowerShell?